City-of-Helsinki/helsinki-paatokset
# PP-411: Old meetings check and calendar fixes

This PR adds changes to the event calendar and some drush commands for debugging meeting and decision data-

**To test**
- Checkout branch, run `make drush-cr`
- Make sure you have some decisions and meetings imported by logging into the container with `make shell` and running:  `drush mim ahjo_decisions:all;drush mim ahjo_decisions:latest;drush mim ahjo_meetings:all;drush mim ahjo_meeetings:latest;drush ap:ud -v`
- Go to: https://helsinki-paatokset.docker.so/fi/kokouskalenteri (if you don't have this page yet, login as admin and create a landing page with the URL alias `/kokouskalenteri`)
  - There should be some meeting visible in the calendar
  - Use the arrows to navigate back to previous months. You should only be able to get back 1 year, after which the previous icon gets disabled.
- Check that the meeting document status script works: `drush ahjo-proxy:check-meeting-records --limit=10 -v`
  - This should produce a list of the latest 10 meetings and their agenda, minutes and document status.
- Check the decision document reset script:
  - Run `drush ap:ud --logic=outdated -v`. This should do nothing since you already ran `drush ap:ud` previously.
  - Run `drush ahjo-proxy:reset-decision-records --limit=10`. This should act on 10 nodes.
  - Run `drush ap:ud --logic=outdated -v` again. This should refetch records for the 10 nodes that were reset.
